OutOfSchool/OutOfSchool.WebApi/Controllers/V1/WorkshopController.cs

```diff
@@ -521,6 +521,29 @@ public async Task<IActionResult> Delete(Guid id)
         return NoContent();
     }
 
+    /// <summary>
+    /// Gets price range of filtered workshops.
+    /// </summary>
+    /// <param name="filter">Filter for workshop entities.</param>
+    /// <returns><see cref="PriceRange"/></returns>
+    /// <response code="200">If the price range was returned.</response>
+    /// <response code="401">If the user is not authorized.</response>
+    /// <response code="403">If the user has no rights to use this method, or sets some properties that are forbidden to change.</response>
+    /// <response code="500">If any server error occures.</response>
+    [ResponseCache(Duration = 300, Location = ResponseCacheLocation.Client, VaryByQueryKeys = new[] { "*" })]
```

Comment:
`Client` means `Cache-Control: private` (it is stored in the browser of each user separately), we need to have it as `Cache-Control: public` - it is cached in the internet and all users can hit this cache. There should be an option for this.